pkg/etw/sample/noop.go

```diff
@@ -0,0 +1,5 @@
+// +build !windows
+
+package main
+
+func main() {}
```

Comment:
just curious, but was this line actually needed to build with Bazel? Does it explicitly need a main function in the package?

Reply:
Unfortunately, yes. Here's the output when I remove
```go
func main() {}
```

```sh
runtime.main_main·f: function main is undeclared in the main package
```

Reply:
Seems to be the case with Go build as well
```sh
go build ./wim/validate
# github.com/Microsoft/go-winio/wim/validate
runtime.main_main·f: function main is undeclared in the main package
```

Reply:
Ah, okay!

Reply:
TIL; thanks!